src/js/filters.js

```diff
@@ -299,22 +314,22 @@ $(document).ready(function() {
         ],
         columns: [
             {
-                data: 'hmisID',
+                data: 'id',
                 name: 'hmis_id',
                 title: 'HMIS'
             },
             {
-                data: 'cjID',
+                data: 'dedup_client_id',
```

Comment:
@KClough sounds like we want to undo this change, but there was nothing else that looked like `cjID` in the sample data